tools/extra/coreidle/main.c

```diff
@@ -240,8 +239,9 @@ int main( int argc, char** argv )
 	ON_ERR_GOTO(result, out_exit, "destroying properties object");
 
 out_exit:
+	if (coreidleCmdLine.gbs_data)
+		free(coreidleCmdLine.gbs_data);
```

Comment:
coreidleCmdLine.gbs_data does not get freed in the case of fpga resource not found, such as in line 185.

Reply:
good catch!